content/enterprise/admin/infrastructure/admin.mdx

```diff
@@ -1,16 +1,12 @@
 ---
-page_title: Active/Active - Infrastructure Administration - Terraform Enterprise
+page_title: Infrastructure Administration - Terraform Enterprise
 ---
 
-# Terraform Enterprise Active/Active
+# Terraform Enterprise Admin CLI commands
 
-When your organization requires increased reliability or performance from Terraform Enterprise that your current single application instance cannot provide, it is time to scale to the Active/Active architecture. Please refer to  [Terraform Enterprise Active/Active](/enterprise/install/automated/active-active) for more information on moving to Active/Active.
+The [Active/Active](/enterprise/install/automated/active-active) operational mode disables the Replicated Admin Console. Instead, it provides admin commands to change the configuration, stop the application safely, and produce support bundles. You must use SSH to log in to a node in the Active/Active cluster to run these commands from the command line.
```

Comment:
Just thinking through flow - that last sentence felt a bit wordy. Okay also - should we say "Active/Active and Standalone?  It's both for the latest versions onward, right?  

```suggestion
The [Active/Active](/enterprise/install/automated/active-active) operational mode disables the Replicated Admin Console. Instead, it provides admin CLI commands to change the configuration, stop the application safely, and produce support bundles. You must use SSH to log in to a node in the Active/Active cluster to run these commands.
```

Reply:
For standalone, the Replicated Admin Console is not disabled so I think leaving it just like your suggested change is fine.